packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -71,11 +74,12 @@ function OverflowMenu(props) {
 
   const handleOpenMenu = () => {
     setIsOpen(true);
-    // https://github.com/acl-services/paprika/issues/316
-    // Todo Should focus the first item via an onAfterOpen event callback in popover
-    setTimeout(() => {
-      focusAndSetIndex(0);
-    }, 250);
+
+    if (document.querySelector("html").getAttribute("data-whatinput") === "keyboard") {
```

Comment:
I think you can get the same with `document.querySelector("[data-whatinput]").dataset.whatIput`

Reply:
oh yeah, good idea